spec/support/sortable_shared_examples.rb

```diff
@@ -0,0 +1,109 @@
+
+    condition_one_handle.drag_to(condition_two_handle)
+
+    within("li.sortable-list:nth-of-type(1)") do
+      expect(page).to have_selector("span", text: "Condition 1")
+      expect(page).to have_selector("h2", text: "Title 2")
+    end
+    within("li.sortable-list:nth-of-type(2)") do
+      expect(page).to have_selector("span", text: "Condition 2")
+      expect(page).to have_selector("h2", text: "Title 1")
+    end
+    within("li.sortable-list:nth-of-type(3)") do
+      expect(page).to have_selector("span", text: "Condition 3")
+      expect(page).to have_selector("h2", text: "Title 3")
+    end
+    expect(condition_one.reload.position).to eq(2)
+    expect(condition_two.reload.position).to eq(1)
+    expect(condition_three.reload.position).to eq(3)
+
+    condition_one_handle.drag_to(condition_three_handle)
+
+    within("li.sortable-list:nth-of-type(1)") do
+      expect(page).to have_selector("span", text: "Condition 1")
+      expect(page).to have_selector("h2", text: "Title 2")
+    end
+    within("li.sortable-list:nth-of-type(2)") do
+      expect(page).to have_selector("span", text: "Condition 2")
+      expect(page).to have_selector("h2", text: "Title 3")
+    end
+    within("li.sortable-list:nth-of-type(3)") do
+      expect(page).to have_selector("span", text: "Condition 3")
+      expect(page).to have_selector("h2", text: "Title 1")
+    end
+    expect(condition_one.reload.position).to eq(3)
+    expect(condition_two.reload.position).to eq(1)
+    expect(condition_three.reload.position).to eq(2)
+
+    condition_three_handle.drag_to(condition_two_handle)
+
+    within("li.sortable-list:nth-of-type(1)") do
+      expect(page).to have_selector("span", text: "Condition 1")
+      expect(page).to have_selector("h2", text: "Title 3")
+    end
+    within("li.sortable-list:nth-of-type(2)") do
+      expect(page).to have_selector("span", text: "Condition 2")
+      expect(page).to have_selector("h2", text: "Title 2")
+    end
+    within("li.sortable-list:nth-of-type(3)") do
+      expect(page).to have_selector("span", text: "Condition 3")
+      expect(page).to have_selector("h2", text: "Title 1")
+    end
+    expect(condition_one.reload.position).to eq(3)
+    expect(condition_two.reload.position).to eq(2)
+    expect(condition_three.reload.position).to eq(1)
+
+    click_link "Back"
+    click_link "Add #{class_name.pluralize}"
+
+    within("li.sortable-list:nth-of-type(1)") do
+      expect(page).to have_selector("span", text: "Condition 1")
+      expect(page).to have_selector("h2", text: "Title 3")
+    end
+    within("li.sortable-list:nth-of-type(2)") do
+      expect(page).to have_selector("span", text: "Condition 2")
+      expect(page).to have_selector("h2", text: "Title 2")
+    end
+    within("li.sortable-list:nth-of-type(3)") do
+      expect(page).to have_selector("span", text: "Condition 3")
+      expect(page).to have_selector("h2", text: "Title 1")
+    end
+
+    if condition_set.pre_commencement?
```

Comment:
Not super happy about doing it this way but I don't know how to create a decision notice in a more abstract way. This is something to come back to when we make these work for informative etc too.